ci/prow/config.yaml

```diff
@@ -1804,9 +1763,8 @@ periodics:
       - "--artifacts=$(ARTIFACTS)"
       - "--profile-name=coverage_profile.txt"
       - "--cov-target=."
-      - "--cov-threshold-percentage=80"
-
-- cron: "30 * * * *" # Run every hour and 30 minutes
+      - "--cov-threshold-percentage=50"
```

Comment:
coverage threshold of 50 instead of 80, is this intended?

Reply:
Yes, to keep it in sync with the threshold in the presubmit and postsubmit jobs.